extensions/functions_arithmetic.yaml

```diff
@@ -161,6 +161,36 @@ scalar_functions:
           - value: fp64
           - value: fp64
         return: fp64
+  -
+    name: "negate"
+    description: "Negation of the value"
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i8
+        return: i8
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i16
+        return: i16
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i32
+        return: i32
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: i64
+        return: i64
+      - args:
+          - value: fp32
+        return: fp32
+      - args:
+          - value: fp64
+        return: fp64
   -
     name: "modulus"
     description: "Get the remainder when dividing one value by another."
```

Comment:
Modulus and remainder are two different things, and even then everyone seems to have their own ideas about them; please specify the behavior for negative numbers. The same thing actually applies for integer division: does it round toward negative infinity or zero, and does this depend on the sign of the divisor? Remainder is then fully defined. I personally much prefer the behavior of Python over what C typically does (and I say "typically" because C shies away from actually specifying it and leaves it to the processor/compiler).

Reply:
Uh, oops.

Reply:
heh, all good. I didn't know that Github was _finally_ allowing comments on lines that hadn't been touched (which I've been waiting for for ages)